dataeng/resources/warehouse-transforms-ci.sh

```diff
@@ -16,11 +16,13 @@ aws s3 cp s3://edx-dbt-docs/manifest.json ${WORKSPACE}/manifest
 cd $WORKSPACE/warehouse-transforms
 
 # Pull the origin master code to latest branch which will be used to compare the diff
-git pull origin master:latest
+git pull -f origin master:latest
 
 # Put back the head at PR commit
 git checkout ${ghprbActualCommit}
 
+git rebase latest
```

Comment:
It is better to do re-basing from master before merging it, so that merge conflicts if any can be resolved. If Warehouse Transform user missed to re-base from master the following git diff command might catch the files that are already tested and merged into master and this job will take extra time to build/test those models as well. That is why I have added this git re-base in the job as well, this will help us to optimize we will only run the modified models as a result the PR in question